src/main/java/io/zold/api/Transaction.java

```diff
@@ -77,4 +79,13 @@
      * @return RSA Signature
      */
     String signature();
+
+    @Override
```

Comment:
@victornoel I'm not familiar with `eo-envelope`, do we need to declare these for it to work? If not let's remove them.